build.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+set -eu
+
+echo λλλ get newest PHP CS Fixer version
+php download/php-cs-fixer-v2.phar self-update || curl --silent --fail -L "https://github.com/FriendsOfPHP/PHP-CS-Fixer/releases/download/$(curl --silent --fail -u keradus:5e7538aa415005c606ea68de2bbbade0409b4b8c https://api.github.com/repos/FriendsOfPHP/PHP-CS-Fixer/releases/latest | jq -r .tag_name)/php-cs-fixer.phar" -o download/php-cs-fixer-v2.phar
+./extractor.php | jq .version
+PHP_CS_FIXER_VERSION=$(./extractor.php | jq -r .version.number)
+PHP_CS_FIXER_CODENAME=$(./extractor.php | jq -r .version.codename)
+
+echo λλλ prepare theme.conf and mark newest PHP CS Fixer version
+cp theme/theme.conf.dist theme/theme.conf
+echo "fixer_version_number = ${PHP_CS_FIXER_VERSION}" >> theme/theme.conf
+echo "fixer_version_codename = ${PHP_CS_FIXER_CODENAME}" >> theme/theme.conf
+
+echo λλλ set up initial folders
+rm -rf dist/
+rm -rf src/
+mkdir -p src/
+
+echo λλλ set up index.rst and 404.rst
+cp index.rst src/index.rst
+cat theme/who_is_behind.rst >> src/index.rst
+cp 404.rst src/404.rst
+
+echo λλλ fetch up doc/ folder from git repo
+curl --silent --fail -L "https://github.com/FriendsOfPHP/PHP-CS-Fixer/archive/v${PHP_CS_FIXER_VERSION}.tar.gz" -o src/archive.tar.gz
+tar -zxf src/archive.tar.gz -C src "PHP-CS-Fixer-${PHP_CS_FIXER_VERSION}/doc/"
+mv "src/PHP-CS-Fixer-${PHP_CS_FIXER_VERSION}/doc/" src/doc/
+rmdir "src/PHP-CS-Fixer-${PHP_CS_FIXER_VERSION}/"
+rm src/archive.tar.gz
+
+echo λλλ replace internal links
```

Comment:
I think there should be a way to tell `sphinx` to support cross-file linking out of the box, but I failed to do so. adding that `:doc:` manually is doing the trick